tsrc/cli/push_gitlab.py

```diff
@@ -35,9 +36,29 @@ def __init__(self, query: str) -> None:
         super().__init__("Found more that one user matching query: %s" % self.query)
 
 
-def get_token() -> str:
+class NoGitLabToken(tsrc.Error):
+    def __init__(self) -> None:
+        message = textwrap.dedent(
+            """\
+            Could not find GitLab token in tsrc config file
+            Please check  https://tankerhq.github.io/tsrc/ref/formats/#tsrcyml_format
+            for details\
+            """
+        )
+        super().__init__(message)
+
+
+def get_token() -> Optional[str]:
     config = tsrc.parse_tsrc_config()
-    res = config["auth"]["gitlab"]["token"]  # type: str
+    auth = config.get("auth")
```

Comment:
Do you thing this would be easier to read if you just did:
```
def get_token() -> str:
    config = tsrc.parse_tsrc_config()
    try:
        return config["auth"]["gitlab"]["token"]
    except KeyError:
        raise NoGitLabToken()
```